tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sinks/KinesisSinkTester.java

```diff
@@ -128,8 +130,9 @@ public void stopServiceContainer() {
 
     @Override
     protected LocalStackContainer createSinkService(PulsarCluster cluster) {
-        return new LocalStackContainer(DockerImageName.parse("localstack/localstack:1.0.4"))
-                .withServices(LocalStackContainer.Service.KINESIS);
+        return new LocalStackContainer(DockerImageName.parse("localstack/localstack:4.0.3"))
```

Comment:
can we pass this image version as a config?

Reply:
We could, but I don't think that it's relevant in this PR. There won't be a need to upgrade this on-the-fly since this isn't an application dependency or even a test dependency.